fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -119,7 +119,18 @@ def self.print_table(actions)
     # Lane chooser if user didn't provide a lane
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
-      available = ff.runner.lanes[platform].to_a.reject { |lane| lane.last.is_private }
+      available = []
+
+      if platform
+        available = ff.runner.lanes[platform.to_sym].to_a.reject { |lane| lane.last.is_private }
+      else
+        # If we still have no platform here, try all supported.
+        # This can be the case if no default platform is given, when multiple platforms are available and plain "fastlane" is called.
+        Fastlane::SupportedPlatforms.all.each do |platform|
```

Comment:
Replacing

```
# If we still have no platform here, try all supported.
        # This can be the case if no default platform is given, when multiple platforms are available and plain "fastlane" is called.
        Fastlane::SupportedPlatforms.all.each do |platform|
          available += ff.runner.lanes[platform].to_a.reject { |lane| lane.last.is_private }
        end
```

with 

```
ff.runner.available_lanes(platform)
```

yields

```
➜  fastlanetest bundle exec fastlane
[17:08:04]: -------------------------------------------------
[17:08:04]: --- Step: Verifying required fastlane version ---
[17:08:04]: -------------------------------------------------
[17:08:04]: Your fastlane version 2.17.1 matches the minimum requirement of 2.17.1  ✅

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
bundler: failed to load command: fastlane (/Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/bin/fastlane)
NoMethodError: [!] undefined method `last' for "android android_lane":String
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/lane_manager.rb:136:in `block in choose_lane'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/lane_manager.rb:135:in `each'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/lane_manager.rb:135:in `each_with_index'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/lane_manager.rb:135:in `choose_lane'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/lane_manager.rb:39:in `cruise_lane'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/commands_generator.rb:105:in `block (2 levels) in run'
  /Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:178:in `call'
  /Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:153:in `run'
  /Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/runner.rb:446:in `run_active_command'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
  /Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/delegates.rb:15:in `run!'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/commands_generator.rb:302:in `run'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/fastlane/lib/fastlane/cli_tools_distributor.rb:63:in `take_off'
  /private/var/folders/vg/gf7092154qs10_l0rh6cfwym0000gn/T/tmp.Zzim68IM/fastlane/bin/fastlane:15:in `<top (required)>'
  /Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/bin/fastlane:22:in `load'
  /Users/admin/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/bin/fastlane:22:in `<top (required)>'
➜  fastlanetest bundle exec fastlane
```